src/components/BuildingMeta/BuildingMeta.vue

```diff
@@ -5,6 +5,7 @@
         :delay="0"
         :overflow-padding="4"
         :instant-move="true"
+        :triggers="['hover', 'click']"
```

Comment:
It's getting more and more props specified and they're always the same, so we might think about extracting it to a shorthand component one day, but that's of course completely optional